iclr2017/python/get-acceptance-status.py

```diff
@@ -26,6 +27,12 @@
 acceptances = client.get_notes(invitation='ICLR.cc/2017/conference/-/paper.*/acceptance')
 reviews = client.get_notes(invitation='ICLR.cc/2017/conference/-/paper.*/official/review')
 
+headers = {'User-Agent': 'test-create-script', 'Content-Type': 'application/json',
+           'Authorization': 'Bearer ' + client.token}
+anon_reviewers = requests.get(client.baseurl + '/groups?id=ICLR.cc/2017/conference/paper.*/AnonReviewer.*',
+                              headers=headers)
+area_chairs = requests.get(client.baseurl+'/groups?id=ICLR.cc/2017/conference/paper.*/areachair1.*', headers = headers)
```

Comment:
there are a few cases where we have an areachair2 instead of areachair1. 

https://openreview.net/groups?id=ICLR.cc/2017/conference/paper.*/areachair2

So, this query is not going to give these groups.